Stripe/UITableViewCell+Stripe_Borders.m

```diff
@@ -15,42 +15,42 @@
 @implementation UITableViewCell (Stripe_Borders)
 
 - (UIView *)stp_topBorderView {
-    UIView *view = [self.contentView viewWithTag:STPTableViewCellTopBorderTag];
+    UIView *view = [self viewWithTag:STPTableViewCellTopBorderTag];
     if (!view) {
         view = [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.bounds.size.width, 0.5f)];
         view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleBottomMargin;
         view.tag = STPTableViewCellTopBorderTag;
         view.backgroundColor = self.backgroundColor;
         view.hidden = YES;
         view.accessibilityIdentifier = @"stp_topBorderView";
-        [self.contentView addSubview:view];
+        [self addSubview:view];
     }
     return view;
 }
 
 - (UIView *)stp_bottomBorderView {
-    UIView *view = [self.contentView viewWithTag:STPTableViewCellBottomBorderTag];
+    UIView *view = [self viewWithTag:STPTableViewCellBottomBorderTag];
     if (!view) {
         view = [[UIView alloc] initWithFrame:CGRectMake(0, self.bounds.size.height - 0.5f, self.bounds.size.width, 0.5f)];
         view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleTopMargin;
         view.tag = STPTableViewCellBottomBorderTag;
         view.backgroundColor = self.backgroundColor;
         view.hidden = YES;
         view.accessibilityIdentifier = @"stp_bottomBorderView";
-        [self.contentView addSubview:view];
+        [self addSubview:view];
     }
     return view;
 }
 
 - (UIView *)stp_fakeSeparatorView {
-    UIView *view = [self.contentView viewWithTag:STPTableViewCellFakeSeparatorTag];
+    UIView *view = [self viewWithTag:STPTableViewCellFakeSeparatorTag];
     if (!view) {
         view = [[UIView alloc] initWithFrame:CGRectMake(0, self.bounds.size.height - 0.5f, self.bounds.size.width, 0.5f)];
         view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleTopMargin;
         view.tag = STPTableViewCellFakeSeparatorTag;
         view.backgroundColor = self.backgroundColor;
         view.accessibilityIdentifier = @"stp_fakeSeparatorView";
-        [self.contentView addSubview:view];
+        [self addSubview:view];
```

Comment:
Ended up moving these separators out of the `contentView` so that they don't move rightward when we start the editing state:

![simulator screen shot jul 17 2017 12 09 41 pm](https://user-images.githubusercontent.com/28276156/28285276-f8b088fa-6ae8-11e7-9427-14aa4f55b273.png)